assessments/guide/outline.md

```diff
@@ -0,0 +1,114 @@
+# Outline
+
+First of all, the burden is primarily on the proposing project to
+demonstrate it is secure in a manner that is understandable to the broader
+community.  The [reviewers](security-reviewer.md) will help to assess and probe the design.
+
+The proposing project must provide a written document that describes the
+project and its security.  The document must contain the following
+information, at a minimum. Where security considerations do not fit into
+the outline below, if possible, add a sub-section when possible such that the
+additional content conforms to the general flow of the review.
+
+
+## Metadata
+A table at the top for quick reference information, later used for indexing.
+
+|   |  |
+| -- | -- |
+| Software | A link to the software’s repository.
+| Security Provider | Yes or No. Is the primary function of the project to support the security of an integrating system?
+|||
+
+## Overview
+
+One or two sentences describing the project -- something memorable and accurate
+that distinguishes your project to quickly orient readers who may be
+reviewing multiple assessments.
+
+* Background. Provide information for reviewers who may not be familiar with
+your project's domain or problem area.
+* Goal. The intended goal of the projects including the security guarantees
+the project is meant to provide (e.g., Flibble only allows parties with an
+authorization key to change data it stores)
+* Non-goals.  Non-goals that a reasonable reader of the project’s literature
+could believe may be in scope (e.g., Flibble does not intend to stop a
+party with a key from storing an arbitrarily large amount of data, possibly
+incurring financial cost or overwhelming the servers)
+
+## Intended use
+
+* Target users and use cases. Provide a mapping from [standard personas](../../usecases.md) to the
+nomenclature used in your project docs (which you should then use consistently
+for the remaider of this document). Describe the scenarios in which the project is expected to be used. This must be specific enough to provide context for analysis. For example:
+
+    Flibble can be used in any cloud environment.  Three diverse examples are
+    as follows:
+    1. when a Flibble server is used by legacy servers as a
+    database for salted password hashes.
+    2. a Flibble cloudlet may be run
+    on virtualized fog hardware near smartphone users.
+    3. a Flibble distributed service may serve as a backend for a Notary image registry.)
+* Operation.  A description of the operational aspects of the system, such as
+how keys are likely to be managed and stored.
+
+## Project design
+* Design.  A description of the system design that discusses how it works.
```

Comment:
is there any interest in promoting/recommending formal design specification ,eg TLA+ and such?

Reply:
This seems great if a project has it.  I don't think we want to require this at the outset.